### PR TITLE
Compressed the CDN section a bit

### PR DESCRIPTION
**Pull request by DandelionSprout**
As simple as that, hopefully.

### CODE REVIEW

**Comment by krystian3w** [comment]
Also fix this #4347 if repo like many updated files into one commit

**Comment by mediumkreation** [comment]
resolved #4357

**Comment by krystian3w** [comment]
IMO copression CDN file was marked as won't fix if wasn't copied to own commit

**Comment by DandelionSprout** [comment]
#4357 did not solve it **at all**. It just changed one completely different entry instead: https://github.com/mediumkreation/IndianList/pull/4357/files

File: IndianList/cdn_block.txt [hunk 1]
```diff
@@ -67,28 +67,23 @@
...
 ||googleapis.com/production-prothomalo/contents/uploads/media/2020/03/21/e228ecefda1d4ffaf696fdbf60b12bd4-5e75b7d59efd0.gif
 ||googleapis.com/production-prothomalo/contents/uploads/media/2020/03/22/a53c582bdeb69a2421656d4c824cec7b-5e77374fd1e5a.png
 ||hostitbd.com/wp-content/uploads/2020/02/website-create-by-hostitbd.png
-||i.ibb.co/64NYFCq/Whats-App-Image-2020-03-06-at-2-34-08-PM.jpg
-||i.ibb.co/dPqxFdk/Whats-App-Image-2020-02-05-at-3-45-48-PM.jpg
-||i.ibb.co/HnmRMHv/Whats-App-Image-2020-02-02-at-5-39-22-PM.jpg
-||i.ibb.co/nkk4q0C/Whats-App-Image-2020-02-25-at-5-21-09-PM.jpg
-||i.ibb.co/Wt2cW3r/Whats-App-Image-2020-02-25-at-5-20-30-PM.jpg
-||i.imgur.com/nu4zSbR.jpg
+||i.ibb.co/*/Whats-App-Image-2020-*-at-*.jpg
```
Comment (mediumkreation) [inline IndianList/cdn_block.txt:74]: these can't be simply compressed as many content image files are with the same pattern.

**Comment by mediumkreation** [comment]
can't simply apply he compression as most of them will target content image.
Secondly, the purpose of septate CDN filters with full urls was to simply remove them with some automated system as and when they get deleted (i.e. response with http response code404).

**Comment by DandelionSprout** [comment]
Ah, okay, that explains it. Though it did take me by surprise that `WhatsApp` images could be either blockable or unblockable depending on the site. 😅

**Pull request closed without merging**